modules/mapper-extras/src/yamlRestTest/resources/rest-api-spec/test/rank_feature/10_basic.yml

```diff
@@ -29,6 +29,63 @@ setup:
           pagerank: 100
           url_length: 20
 
+  - do:
```

Comment:
Let's create a separate file for `null_value` tests, and call it `20_null_value.yml`.

We also need to limit this new test only for versions starting from  8.9, but putting this header:

```
  - skip:
      version: " - 8.8.99"
      reason: "null_value parameter was added in 8.9.0"
```